revaturelabs/ProjectForce
# Development

Color injection fixes. To test:

- try to put in a non-hexidecimal value for the color on the Batch Table, make sure it rejects it
- make sure there is a validation rule on the Color__c field on Training__c that checks for hexidecimal regex
